api/controller/ReportController.py

```diff
@@ -0,0 +1,77 @@
+from flask import Response, request, render_template
+from api.models.report import Report
+from mongoengine.errors import FieldDoesNotExist, NotUniqueError, DoesNotExist, ValidationError, InvalidQueryError
+import pry
+
+class ReportController():
+    # GET single
+    def get_report(self, id):
+        report = Report.objects.get(id=id)
+        return {
+            "data": {
+                "type": str(report),
+                "id": str(report.id),
+                "attributes": {
+                    "input": {
+                        "salary": report.salary,
+                        "zipcode": report.zipcode,
+                        "credit_score": report.credit_score,
+                        "monthly_debt": report.monthly_debt,
+                        "downpayment_savings": report.downpayment_savings,
+                        "downpayment_percentage": report.downpayment_percentage,
+                        "rent": report.rent,
+                        "goal_principal": report.goal_principal,
+
+                    },
+                    "output": {}
+                }
+            }
+        }, 200
+
```

Comment:
The error handling is on another branch.

Reply:
Lovely. I know this is a "temporary" deployment
We will continue working on individual features, like error paths, new models, and finalizing report functions and merge those to dev